models/User.js

```diff
@@ -5,9 +5,9 @@ const userSchema = new Schema({
 	isOwner: Boolean,
 	isPlayer: Boolean,
 	isAdmin: Boolean,
-	ign: String,
 	elo: Number,
```

Comment:
If we're allowing multiple accounts could they not have different elos? Then there's the same issue with the 'team' field too. Thoughts on removing these here and just keeping them in the playerSchema? Or at least 'elo' and keeping it one team per account if that's what you had in mind.